src/work/SimpleDAO.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title SimpleDAO
+ * @dev A simplified version of a DAO that allows members to create proposals,
+ */
+contract SimpleDAO {
+    struct Proposal {
+        string description;
```

Comment:
long strings are expensive to store. Consider doing this offchain

Reply:
added constraint
    uint public constant MAX_DESCRIPTION_LENGTH = 200; // Maximum length for proposal descriptions